android/src/main/java/io/sentry/react/RNSentryModule.java

```diff
@@ -130,6 +130,12 @@ public void initNativeSdk(final ReadableMap rnOptions, Promise promise) {
             if (rnOptions.hasKey("sendDefaultPii")) {
                 options.setSendDefaultPii(rnOptions.getBoolean("sendDefaultPii"));
             }
+            if (rnOptions.hasKey("transportOptions")) {
+                final ReadableMap transportOptions = rnOptions.getMap("transportOptions");
+                if (transportOptions.hasKey("bufferSize")) {
+                    options.setMaxQueueSize(rnOptions.getInt("bufferSize"));
+                }
+            }
```

Comment:
H: This should be called `maxQueueSize` and it should be a high-level property, not under `transportOptions`.